app/components/primer/alpha/action_list.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+module Primer
+  module Alpha
+    # An `ActionList` is a TBD
+    class ActionList < Primer::Component
+      DEFAULT_ROLE = :list
+      DEFAULT_TAG = :ul
+
+      DEFAULT_SCHEME = :full
+      SCHEME_MAPPINGS = {
+        DEFAULT_SCHEME => nil,
+        :inset => "ActionListWrap--inset"
+      }.freeze
+      SCHEME_OPTIONS = SCHEME_MAPPINGS.keys.freeze
+
+      def self.custom_element_name
+        @custom_element_name ||= name.split("::").last.underscore.dasherize
+      end
+
+      def custom_element_name
+        self.class.custom_element_name
+      end
+
+      renders_one :heading, lambda { |**system_arguments|
+        Heading.new(section_id: @id, **system_arguments)
+      }
+
+      renders_many :items, lambda { |**system_arguments|
+        build_item(**system_arguments).tap do |item|
+          will_add_item(item)
+        end
+      }
+
+      renders_many :lists, lambda { |**system_arguments|
+        build_list(child: true, **system_arguments).tap do |group|
+          will_add_list(group)
+        end
+      }
+
+      def initialize(
+        tag: DEFAULT_TAG,
+        role: DEFAULT_ROLE,
+        item_classes: nil,
+        scheme: DEFAULT_SCHEME,
+        show_dividers: false,
+        child: false,
```

Comment:
@joelhawksley should we call this subGroup? Thats what I had in the API doc but I'm not sure where it came from. I don't have a strong opinion either way.